src/tests/functional/plugin/gpu/remote_blob_tests/remote_blob_helpers.hpp

```diff
@@ -89,12 +89,15 @@ struct OpenCL {
                 cl::Platform platform = cl::Platform(id);
                 std::vector<cl::Device> devices;
                 platform.getDevices(CL_DEVICE_TYPE_GPU, &devices);
+                unsigned int count = 0;
                 for (auto& d : devices) {
                     if (refVendorID == d.getInfo<CL_DEVICE_VENDOR_ID>()) {
                         _device = d;
                         _context = cl::Context(_device);
                         _platform = id;
-                        break;
+                        if (count == index)
+                            break;
+                        count++;
                     }
```

Comment:
@vladimir-paramuzov  @sshlyapn can  you help review this change when trying to get multiple ocl instances for multiple gpus?